### DISCUSSION TITLE
Transaction History

### DISCUSSION THREAD

**Discussion by JDEvolutionist** [iotaledger/firefly#7550] [Help & Support]
How do I get a CSV transaction history file using the latest Firefly App update for MacBook. The old App. made this fairly easy but the latest update does not seem to allow creation of a CSV file. Backing up the account talks in terms of transaction history in stronghold but there does not seem a way of accessing it.

**Comment by JDEvolutionist** [comment]
I need the CSV file for linking my  IOTA account data with Koinly. Have tried doing this with the public key but no useful data is transferred. After syncing with Koinly it come back to say 0 updates and 6 transactions skipped but no transactions show for the wallet on Koinly.

**Reply by DeafEyeJedi** [reply]
Any fix for this yet? I need to import CSV file from FireFly to fill into Koinly.io properly.

**Reply by begonaalvarezd** [reply]
You can use the explorer for this, we have the feature there, firefly will not have the CSV export because the transaction of the wallet is not 100% accurate due to node pruning. If you need an accurate tx history and an export feature, the explorer is the place:
- Stardust (post merge October 4 2023): https://explorer.iota.org/mainnet
- Chrysalis (pre merge): https://explorer.iota.org/chrysalis-mainnet
